.gitignore

```diff
@@ -1,3 +1,4 @@
 build
 composer.lock
 vendor
+.idea/
```

Comment:
Esse é um arquivo do seu editor? Se for, é melhor colocar no `git config --global` em vez de adicionar no projeto.